README.md

```diff
@@ -182,7 +182,7 @@ WAGTAILLOCALIZE_MACHINE_TRANSLATOR = {
 Django's `LocaleMiddleware` detects a user's browser language and forwards them to the most appropriate language
 version of the website.
 
-To enable it, insert `'django.middleware.locale.LocaleMiddleware'` into the middleware setting
+To enable it, insert `"django.middleware.locale.LocaleMiddleware"` into the middleware setting
```

Comment:
Note here and in lots of other places – by default the generated site uses single quotes, so those examples differ from `wagtail start` output. Here I switched to double for consistency with the code below.